.github/workflows/publish-v2.yml

```diff
@@ -142,4 +142,13 @@ jobs:
       - name: Remove git tag
         uses : ./.github/workflows/remove-git-tag
         with:
-          TAG_NAME: v${{ env.CLEAN_PACKAGE_NAME }}
\ No newline at end of file
+          TAG_NAME: v${{ env.CLEAN_PACKAGE_NAME }}
+      - name: Merge to master
+        if: env.CHANNEL == 'ga' # Only for GA releases
+        uses: devmasx/merge-branch@a1752b9ba42bb417ec19be7dc974e2faf77d3ef2 # v1.3.1
```

Comment:
1.4.0 is the latest release

Reply:
Version updated to devmasx/merge-branch@854d3ac71ed1e9deb668e0074781b81fdd6e771f # v1.4.0